cfme/tests/storage/test_volume.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+import pytest
+from widgetastic.exceptions import NoSuchElementException
+
+from cfme import test_requirements
+from cfme.cloud.provider import CloudProvider
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.storage.volume import Volume
+
+from utils import testgen
+from utils.appliance.implementations.ui import navigate_to
+
+
+pytest_generate_tests = testgen.generate([CloudProvider])
```

Comment:
JFYI, I've recently struggled with the following case. I had to exclude SCVMM from InfraProviders.
I managed to get around it using - [the following construction](https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/infrastructure/test_timelines.py#L25)
It definitely makes sense for you only if you decide to extend volume tests.